src/utils/searchUtils.ts

```diff
@@ -287,6 +302,11 @@ export function mapQueryToSearchParams({
     contributor,
     title,
     subject,
+    // TODO: this is a "catch-all" for journal title and standard number
```

Comment:
I like the approach of using a catch all here, though I'm not sure if we'll need more explicit typing down the road. Should be good for now though.